src/index.html

```diff
@@ -9,6 +9,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="moyo-container">
+      <header class="main-header">
+        <img src="images/logo.png" alt="moyo-logo1"
+         class="moyo-logo1">
```

Comment:
```suggestion
        <img 
          src="images/logo.png" 
          alt="moyo-logo1"
          class="moyo-logo1"
        >
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/4e27ecad-49a5-4bba-bb36-e623b21a5c91)